moped-editor/src/queries/project.js

```diff
@@ -48,10 +48,12 @@ export const SUMMARY_QUERY = gql`
       moped_project {
         project_name
       }
-      moped_proj_components(where: { is_deleted: { _eq: false } }) {
-        moped_proj_features(where: { is_deleted: { _eq: false } }) {
-          feature_id
-          feature
+      moped_proj_components(
+        where: { is_deleted: { _eq: false }, feature_signals: {} }
+      ) {
+        feature_signals {
```

Comment:
Does this part of the query need to filter `is_deleted = false`? That way deleted signals don't end up here. Makes me wonder how that ties into the Knack record as well. I'm not super well-versed in the Knack <> Moped features so these PRs have been great to learn from!

Reply:
Just to expand on that in case it isn't clear. Is it "once a signal project, always a signal project"? What happens if someone creates a signal project, removes the signal component, and then the Knack record remains behind? Is that okay? Just wondering how it is supposed to work.

Reply:
👆 i'm going to copy this note into the code